src/components/divs/ContactDiv.tsx

```diff
@@ -0,0 +1,18 @@
+type Props = {
+  name: string
+  position: string
+  email: string
+  phone: string
+}
+
+export const ContactDiv = ({name, position, email, phone}: Props) => {
+
+  return(
+    <div className="flex flex-col text-xl py-2">
+      <div className="font-bold uppercase">{name}</div>
```

Comment:
instead of using div elements, we could use p elements to have better html semantics